sympy/printing/tests/test_glsl.py

```diff
@@ -834,35 +878,39 @@ def test_misc_mats():
 [12, 13, 14, 15],
 [16, 17, 18, 19]])
 
-    gl = '''float[20](
+    gl = textwrap.dedent('''\
```

Comment:
If you're going to change _all_ these tests, it would make more sense to split this into two PRs:

* One that makes these stylistic changes only
* One that introduces the new GLSL features.

Reply:
I've reverted those extra changes.